### DISCUSSION TITLE
separate PyPlot window from Pluto cell

### DISCUSSION THREAD

**Discussion by RoyCCWang** [fonsp/Pluto.jl#1519] [Questions about using Pluto] [answered]
Is there a way to force Pluto to create a new separate PyPlot window that has the usual PyPlot interactive buttons (e.g., zoom, pan, etc) instead of having PyPlot embed in Pluto?

An example of the separate PyPlot window that I'm talking about can be generated if one plots from the Julia REPL:
```
import PyPlot
t = LinRange(-3,3,100)
PyPlot.plot(t,sin.(t))
```

I understand one can custom program interactive behaviour into Pluto, but the default PyPlot interactive buttons are enough for my usage.

I have tried display() in the Pluto cell, and it isn't working on my machine. My code:
```
fig = PyPlot.figure(1)
t = LinRange(-3,3,100)
PyPlot.plot(t,sin.(t))
PyPlot.display(fig)
```
I'm using Julia v1.6.2, Pluto v0.16.0, Fedora 34.

**Comment by ctrekker** [comment, marked as answer]
I believe you should be able to display the PyPlot window by calling `PyPlot.show()` in a Pluto cell.

![image](https://user-images.githubusercontent.com/22894011/135165909-98d9c972-7c15-42e6-9dfe-22d7f027149d.png)
